Module_8/Q8.js

```diff
@@ -0,0 +1,47 @@
+const tasks = [
+  function () {
+    return new Promise((resolve) => {
+      setTimeout(() => {
+        resolve("Chopping done");
+      }, 1000);
+    });
+  },
+  function () {
+    return new Promise((resolve) => {
+      setTimeout(() => {
+        resolve("Baking done");
+      }, 2000);
+    });
+  },
+  function () {
+    return new Promise((resolve) => {
+      setTimeout(() => {
+        resolve("Serving done");
+      }, 1000);
+    });
+  },
+  function () {
+    return new Promise((resolve) => {
+      setTimeout(() => {
+        resolve("Eating done");
+      }, 1000);
+    });
+  }
+];
+
+async function dotask(tasks) {
+  
+  const res = [];
+  for (task of tasks) {
+    const p = await task();
```

Comment:
Good! you can also try without using async await. It will require a bit more efforts